src/SvgIcon/SvgIcon.spec.js

```diff
@@ -0,0 +1,45 @@
+// @flow weak
+/* eslint-env mocha */
+
+import React from 'react';
+import { assert } from 'chai';
+import { createShallowWithContext } from 'test/utils';
+import SvgIcon, { styleSheet } from './SvgIcon';
+
+describe('<SvgIcon />', () => {
+  let shallow;
+  let classes;
+  let path;
+
+  before(() => {
+    shallow = createShallowWithContext();
+    classes = shallow.context.styleManager.render(styleSheet);
+    path = <path d="M10 20v-6h4v6h5v-8h3L12 3 2 12h3v8z" />;
+  });
+
+  it('renders children by default', () => {
+    const wrapper = shallow(
+      <SvgIcon>{path}</SvgIcon>,
+    );
+    assert.ok(wrapper.contains(path), 'should contain the children');
```

Comment:
`contains` returns a Boolean ([spec](http://airbnb.io/enzyme/docs/api/ShallowWrapper/contains.html)) but `ok` only asserts that the object is truthy.
What about using
```jsx
assert.strictEqual(wrapper.contains(path), true, 'should contain the children');
```
We tend to use `strictEqual` everywhere as conceptually simpler to understand. And simplicity is critical with tests.